frontend/test/metabase-lib/lib/queries/StructuredQuery.unit.spec.js

```diff
@@ -28,6 +30,36 @@ function makeQueryWithAggregation(agg) {
   return makeQuery({ aggregation: [agg] });
 }
 
+function makeQueryWithoutNumericFields() {
```

Comment:
I'm pretty new to testing `Question` and `StructuredQuery` in general, this might not be ideal, but it does the job for now. Please feel free to suggest better alternatives.

I need the question to rely on the saved question, if we only rely on one of the sample table, they already have almost all field types which results in every aggregations being shown.

This setup create a question based on `Orders` table which only has 2 fields selected (ID and created_at), so there's no numeric field at this point.